components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/JWTTokenIssuer.java

```diff
@@ -725,4 +730,28 @@ private JWTClaimsSet handleTokenBinding(JWTClaimsSet.Builder jwtClaimsSetBuilder
         }
         return jwtClaimsSetBuilder.build();
     }
+
+    /**
+     * Set tenant domain of user to the JWT token's realm claim if signed with user tenant.
+     * @param tenantDomain
+     * @param jwtClaimsSet
+     * @return
+     */
+    private JWTClaimsSet setSignerRealm(String tenantDomain, JWTClaimsSet jwtClaimsSet) {
+
+        Map<String, String> realm = new HashMap<>();
+        if (!OAuthServerConfiguration.getInstance().getUseSPTenantDomainValue()) {
+            realm.put(OAuthConstants.OIDCClaims.SIGNED_KEY_TENANT, tenantDomain);
+        }
+        if (realm.size() > 0) {
+            if (log.isDebugEnabled()) {
+                log.debug("Setting authorized user tenant domain : " + tenantDomain +
+                        " used to for token signature to the 'realm' claim of jwt token");
```

Comment:
Grammatical error in the sentence please correct it "used to for"

Reply:
Fixed. Thanks